isotovideo

```diff
@@ -242,7 +242,7 @@ checkout_git_repo_and_branch('CASEDIR');
 #
 # This allows further structuring the test distribution collections with
 # multiple distributions or flavors in one repository.
-$bmwqemu::vars{PRODUCTDIR} ||= $bmwqemu::vars{CASEDIR};
+$bmwqemu::vars{PRODUCTDIR} ||= $bmwqemu::vars{DISTRI} ? $bmwqemu::vars{CASEDIR} . '/products/' . $bmwqemu::vars{DISTRI} : $bmwqemu::vars{CASEDIR};
```

Comment:
I doubt this is correct. Having the variable "DISTRI" defined does not mean that we use a product dir